riff-raff.yaml

```diff
@@ -26,3 +26,4 @@ deployments:
         dependencies: [ dotcom-components-cloudformation ]
         parameters:
             bucket: membership-dist
+            asgMigrationInProgress: true
```

Comment:
Hi,

[This deploy](https://riffraff.gutools.co.uk/deployment/view/1b0cfd32-f786-44b4-9b58-69e9b2ef16c6) of this branch is hanging. It's unclear why and we're looking into it. I was wondering what was the reason behind this change?

Reply:
we shipped this change while part way through a migration to cdk, so we had 2 ASGs. @jacobwinch might have more info...

Reply:
Ah, forgot that you're mid-way through migrating to CDK! https://github.com/guardian/riff-raff/pull/632 adds a bit of context for this Riff-Raff property.

I'd recommend keeping PRs to a single concern however (and rebasing to stay up to date) as they are easier to reason about.